src/foam/dao/SequenceNumberDAO.js

```diff
@@ -25,6 +25,10 @@ foam.CLASS({
 
   documentation: 'DAO Decorator which sets a specified property\'s value with an auto-increment sequence number on DAO.put() if the value is set to the default value.',
 
+  javaImports: [
```

Comment:
I don't think we want to change SequenceNumberDAO. It should remain as is (generating sequence numbers), and instead we should create a similar to the GUIDDAO.